geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/RegionFunctionArgs.java

```diff
@@ -459,81 +447,72 @@ public EvictionAttributes getEvictionAttributes() {
     return (RegionAttributes<K, V>) this.regionAttributes;
   }
 
+  /**
+   * the difference between this and ExpirationAttributes is that this allows time and action to be
+   * null
+   */
   public static class ExpirationAttrs implements Serializable {
```

Comment:
This class gets a pretty significant rework.  I'd love to see it extracted from `RegionFunctionArgs` while we're at it, since it doesn't depend on any otherwise-inaccessible field or member.

Ditto that while we're about it for `PartitionArgs` and `EvictionAttrs`, although again, maybe this is enough of a refactor to warrant a separate ticket.  But as it stands, 40% of this class is just container fluff.

Reply:
agree, another refactor effort.